stylesheets/_component.search.scss

```diff
@@ -11,7 +11,7 @@
     }
 
     @include respond-min($screen-desktop) {
-        left: 95px;
+        left: 105px;
```

Comment:
It seems to me like we should utilising variables here, if the search position is relative the nav width, can't we reference the `$nav-primary-width` and add left hand padding/margin value?

Reply:
Good spot, pushed a fix